src/pipeline.cc

```diff
@@ -716,8 +724,7 @@ class PipelineWorker : public Napi::AsyncWorker {
             baton->channels = std::min(baton->channels, 3);
           }
         } else if (baton->formatOut == "png" || (baton->formatOut == "input" &&
-          (inputImageType == sharp::ImageType::PNG || inputImageType == sharp::ImageType::GIF ||
-            inputImageType == sharp::ImageType::SVG))) {
+          (inputImageType == sharp::ImageType::PNG || inputImageType == sharp::ImageType::SVG))) {
```

Comment:
This is probably a breaking change. The same for *file output*.

Reply:
Could this be based on a runtime check of whether *magick output is supported?